tasks.py

```diff
@@ -22,6 +22,11 @@ def test(ctx, watch=False, last_failing=False):
     if last_failing:
         args.append('--lf')
     args.append('tests')
+    try:
```

Comment:
This try-except should not be necessary, since we're installing apispec-webframeworks in dev requirements.